sympy/solvers/ode.py

```diff
@@ -287,6 +287,7 @@
 #: ``best``, and ``all_Integral`` meta-hints should not be included in this
 #: list, but ``_best`` and ``_Integral`` hints should be included.
 allhints = (
+    "order_reducing_substitution",
```

Comment:
What do you think about the name `reducible` which follows the descriptive names already present?

Reply:
Seems reasonable. The hint names are a bit of hodge-podge between descriptions of types of ODEs and methods for solving ODEs.